src/lib/components/workflow/child-workflows-table.svelte

```diff
@@ -58,7 +58,7 @@
       <th>{translate('workflows.child-id')}</th>
       <th>{translate('workflows.child-run-id')}</th>
     </TableHeaderRow>
-    {#each visibleItems as child (child.runId)}
+    {#each visibleItems as child}
```

Comment:
Can there be duplicate runIds? Or is this to prevent the same workflow shown twice?

Reply:
> Also when I run samples-typescript / child-workflows, I don't get any errors (not on this branch)

Running that sample was causing this error ⬇️ Which was because multiple children had the same `runId`.

![Screenshot 2023-10-26 at 5 06 32 PM](https://github.com/temporalio/ui/assets/15069288/a7e3f31b-dfde-411f-94be-368057a286ee)